webpack/src/index.js

```diff
@@ -0,0 +1,16 @@
+// import _ from 'lodash';
+import './style.css';
+
+const todo = [
+  { description: 'Wash the dishes', isCompleted: false, index: 0 },
+  { description: 'Complete To Do List Project', isCompleted: false, index: 1 },
+];
+
+const todoList = () => {
+  let todoListContent = '';
+  todo.forEach((item) => {
+    todoListContent += `<li class="task"><input class="task-check" type="checkbox"><span class="list">${item.description}</span><i class='fa fa-ellipsis-v' style="float: right;"></i></li>`;
+  });
+  document.querySelector('.task-box').innerHTML = todoListContent;
```

Comment:
- [ ] We can improve this design to be more similar to the template given. Kindly add some top and bottom padding to the tasks as we have in the template.

  Your Implementation | Template Design
  :---:|:---:
  ![one-1](https://user-images.githubusercontent.com/81468675/183694570-bc1dce94-c649-458c-917f-570e7f46be8b.jpg) | ![two-2](https://user-images.githubusercontent.com/81468675/183694546-de679438-5db0-4f01-98c9-e2a93611354e.jpg)